### PR TITLE
Updated javadoc for Presenter, PresenterWidget and HasSlots

### PR DESCRIPTION
**Pull request by Elrhino**

### CODE REVIEW

File: gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/HasSlots.java [hunk 1]
```diff
@@ -151,10 +151,9 @@
      * <p/>
      * For more details on slots, see {@link HasSlots}.
      *
-     * @param slot    An opaque object identifying which slot this content is being
-     *                added into.
-     * @param content The content, a {@link PresenterWidget}. Passing {@code null}
-     *                will not add anything.
+     * @param slot  The slot this content is being added into.
```
Comment (Pacane) [inline gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/HasSlots.java]: This sentence is rather unclear.

**Comment by Pacane** [comment]
LGTM

File: gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/HasSlots.java [hunk 2]
```diff
@@ -151,10 +151,9 @@
      * <p/>
      * For more details on slots, see {@link HasSlots}.
      *
-     * @param slot    An opaque object identifying which slot this content is being
-     *                added into.
-     * @param content The content, a {@link PresenterWidget}. Passing {@code null}
-     *                will not add anything.
+     * @param slot  The slot from which the content will be added.
```
Comment (Pacane) [inline gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/HasSlots.java]: `the content will be added into this slot.` maybe?

**Comment by meriouma** [comment]
LGTM

File: gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java [hunk 3]
```diff
@@ -249,6 +264,18 @@ public void clearSlot(Object slot) {
...
+     * {@link com.gwtplatform.mvp.client.proxy.ResetPresentersEvent} is fired.
+     * The attached {@link View} should manage this slot when its
+     * {@link View#setInSlot(Object, com.google.gwt.user.client.ui.Widget)} is called. It should also clear
+     * the slot when the {@link View#setInSlot(Object, com.google.gwt.user.client.ui.Widget)} method is
+     * called with {@code null} as a parameter.
+     * <p/>
+     * For more details on slots, see {@link HasSlots}.
+     *
+     * @param slot Specific slot to clear.
+     */
```
Comment (Chris-V) [inline gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/PresenterWidget.java]: Is this copied from `HasSlot`? The generated javadoc should duplicate the description already.

**Pull request merged**